src/views/DVA/verification/verification.vue

```diff
@@ -1373,6 +1523,72 @@ export default {
       this.$LIPS(false)
       this.$scrollToTop()
     },
+    async saveNewDoc(document, modal) {
+      this.storeURL = "/api/new_document"
+      this.$LIPS(true)
+      this.verificationData = {
+                customer_id: this.customer.id,
+                document: this.form[document],
+                name: document
+            }
+      // this.form.document = document
+      const form = toMulipartedForm(this.verificationData, "edit")
+      await post(this.storeURL, form)
+        .then(({ data }) => {
+          this.updateView(data.response)
+          log(
```

Comment:
what does this log do?

Reply:
I met this function before I worked on the task, it's showing that I added it because I already removed it and committed and added it back due to some concerns that Babajide raised, however, I think it logs the document name and Id

Reply:
Log it where?

Reply:
The Log is imported from utility, It's a function outside the file which was imported

Reply:
Not an error log or console log

Reply:
What does it do exactly?